lib/capistrano/tasks/maintenance.rake

```diff
@@ -83,14 +83,13 @@ namespace :maintenance do
       info ""
       info "Add the \e[35mfollowing configuration\e[0m to your Apache \e[35m.htaccess\e[0m file:"
       info "=============================="
-      info "<LocationMatch \"\\.(css|gif|ico|jpe?g|js|png|svg|webp|webm)$\">"
-      info "    ErrorDocument 503 \"Service unavailable.\""
-      info "</LocationMatch>"
+      info "ErrorDocument 503 \"Service unavailable.\""
       info "<IfModule mod_rewrite.c>"
       info "    <If \"! %{REQUEST_URI} =~ /\\.(css|gif|ico|jpe?g|js|png|svg|webp|webm)$/i && -f '%{DOCUMENT_ROOT}/#{fetch(:maintenance_basename)}'\">"
       info "        ErrorDocument 503 /#{fetch(:maintenance_basename)}"
       info "    </If>"
       info "    RewriteEngine On"
+      info "    RewriteOptions InheritDownBefore"
```

Comment:
the magic instruction that allow us to use this config into a parent `.htaccess`